distribution/scripting.md

```diff
@@ -38,6 +40,7 @@ registerPlugin({
     version: '1.0',
     authors: ['Your Name'],
     type: 'remote',
+    license: 'Unlicense',
```

Comment:
```suggestion
    license: 'Unlicensed',
```
?

Reply:
I was really going for the [Unlicense](https://choosealicense.com/licenses/unlicense/), but I changed the example to `MIT` as per Michael's suggestion

Reply:
Ah, I wasn't familiar with that one. I think we shouldn't specify any licence by default, though. Instead, force the plugin author to specify one — and have the TypeScript transpiler throw an error if a non-empty string is provided, or some such.

Reply:
No. This refers to "The Unlicense".

Reply:
Personally, I think providing a little bit of guidance would be OK. Perhaps we should:
- throw an error if the field is missing or empty
- have it empty by default, so that everyone *has* to pick one
- suggest MIT in the .md file if the user is unsure

What do you think?